src/C2HS/Gen/Monad.hs

```diff
@@ -200,7 +200,7 @@ transTabToTransFun prefx rprefx (CHSTrans _2Case chgCase table omits) =
 --   the second string is for function results; this distinction is necessary
 --   as 'ForeignPtr's cannot be returned by a foreign function; the
 --   restriction on function result types is only for the actual result, not
---   for type arguments to parameterised pointer types, i.e., it holds for @res@
+--   for type arguments to parametrised pointer types, i.e., it holds for @res@
```

Comment:
Huh, I've never seen it spelled that way before and it does look like both are [valid](https://en.wiktionary.org/wiki/parametrise). I'll defer to you.

Reply:
> spelled that way before

in the cabal repo "parameterized" shows up